### ISSUE TITLE
Clone types are broken

### ISSUE THREAD

**Issue by romainmenke** [postcss/postcss#1863]
```js
	root.walkDecls((decl) => {
		const parent = decl.parent;
		if (!parent) {
			return;
		}

		const parentClone = parent.clone();

		parentClone.each((node) => {});
```

This used to work but now gives warnings.

None of the container typings are present when cloning a `Container`.

----

- `Property 'removeAll' does not exist on type 'Node'. Did you mean 'remove'?`
- `Property 'append' does not exist on type 'Node'.`
- `Property 'each' does not exist on type 'Node'.`
- `Parameter 'node' implicitly has an 'any' type.` (when trying to use `clone.each((node) => {})`)

**Comment by ai** [comment]
Fixed a55c3e28

**Comment by ai** [comment]
Released in 8.4.27

**Comment by romainmenke** [comment]
Awesome 🎉 

All the issues are gone, thank you for the super fast response and fix 🙇
